docs/user/citing.md

```diff
@@ -31,15 +31,20 @@ Possible services:
 author={Schmidt, M.T. and Detwiller, M.H. and
```

Comment:
Should we get rid of this with Zenodo having it?

Reply:
Yeah, I don't think we need it here if we add the zenodo link, it's just another thing we would need to remember to update for each release.